xbmc/cores/AudioEngine/Sinks/windows/AESinkFactoryWin.cpp

```diff
@@ -45,7 +45,7 @@ const char *WASAPIErrToStr(HRESULT err)
     ERRTOSTR(E_OUTOFMEMORY);
   default: break;
   }
-  return NULL;
+  return "Undefined";
```

Comment:
When error code (HRESULT) is not none of defined returns NULL pointer C string that ends in the logs...